YML.md

```diff
@@ -1,12 +1,21 @@
 # About `.yml` files
 
-The `recipes/meta` directory in this repository contains `.yml` files.
+The `recipes/meta` directory in this repository contains `.yml` files. These are used by the `Recipe` to simplify the creation of AppImages, especially by using pre-built binaries from Debian packages, both to save time for creating and building an AppImage.
```

Comment:
It was a little redundant, while restructuring I moved the information to the paragraph in line 14.